### PR TITLE
Adding refresh option to cluster properties page

### PR DESCRIPTION
**Pull request by tejhan**
This PR adds a refresh button to the cluster properties page.
 - Closes issue #291 
 - Minor UI tweaks
 - Telemetry for refresh actions
 
refresh-ezgif com-video-to-gif-converter

.vsix for testing -> [vscode-aks-tools-1.5.3-refresh-test.zip](https://github.com/user-attachments/files/18099981/vscode-aks-tools-1.5.3-refresh-test.zip)

### CODE REVIEW

**Review by Tatsinnit — APPROVED** [review]
❤️❤️☕️ Looks good to me!! Thank you so much!! Let’s get this in.

**Review by ReinierCC — APPROVED** [review]
Looks good!

File: webview-ui/src/ClusterProperties/ClusterProperties.module.css [hunk 1]
```diff
@@ -74,3 +76,28 @@ th {
...
+}
+
+.refreshButton:hover {
+    background: var(--editor-selectionHighlightBackground, rgba(58, 58, 58, 0.4));
+    cursor: pointer;
+}
+
+.refreshButton i {
+    position: relative;
+    top: 6%;
```
Comment (hsubramanianaks) [inline webview-ui/src/ClusterProperties/ClusterProperties.module.css:101]: can we keep it consistent with rem please?

**Review by hsubramanianaks — DISMISSED** [review]
looks good, just a minor suggestion. Thank you.

**Review by Tatsinnit — APPROVED** [review]

**Review by hsubramanianaks — APPROVED** [review]

**Pull request merged**